microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Fdisk, Mount, Parted, Xfstests
+from lisa.tools.fdisk import FileSystem
+
+
+def _configure_disk(
+    node: Node,
+    disk_name: str,
+    first_mountpoint: str = "/root/test",
+    second_mountpoint: str = "/root/scratch",
+    file_system: FileSystem = FileSystem.xfs,
+) -> None:
+    node.execute(f"rm -r {first_mountpoint}", sudo=True)
+    node.execute(f"rm -r {second_mountpoint}", sudo=True)
+    mount = node.tools[Mount]
+    first_disk = f"{disk_name}1"
+    second_disk = f"{disk_name}2"
+    fdisk = node.tools[Fdisk]
+    parted = node.tools[Parted]
+
+    mount.umount(first_disk, first_mountpoint)
+    mount.umount(second_disk, second_mountpoint)
+
+    parted.make_label(disk_name)
+    parted.make_partition(disk_name, "primary", "1", "50%")
```

Comment:
Both parted and fdisk has the `make_partition` method, but it looks the functionality is different. So one of the name may be inaccurate, it's not "make partition", it may make something others like volume?